Tests/SentryTests/SentryHubTests.swift

```diff
@@ -811,11 +811,12 @@ class SentryHubTests: XCTestCase {
         XCTAssertEqual(firstImage["type"] as! String, "macho")
         
         let sampledProfile = profile["sampled_profile"] as! [String: Any]
-        let threadMetadata = sampledProfile["thread_metadata"] as! [String: Any]
+        let threadMetadata = sampledProfile["thread_metadata"] as! [String: [String: Any]]
         let queueMetadata = sampledProfile["queue_metadata"] as! [String: Any]
         
         XCTAssertFalse(threadMetadata.isEmpty)
-        XCTAssertGreaterThan((threadMetadata.first?.value as! [String: Any])["priority"] as! Int, 0)
+        XCTAssertGreaterThan(threadMetadata.first?.value["priority"] as! Int, 0)
+        XCTAssertFalse(threadMetadata.values.filter { $0["is_main_thread"] as? Bool == true }.isEmpty)
```

Comment:
Can we have a test where the main thread shows up in the list?

Reply:
`XCTAssertFalse(threadMetadata.values.filter { $0["is_main_thread"] as? Bool == true }.isEmpty)` is checking whether there is something in the list that is marked as being the main thread, did you mean something else?